static/js/compose.js

```diff
@@ -87,6 +87,11 @@ export function clear_preview_area() {
     $("#compose .preview_content").empty();
     $("#compose .markdown_preview").show();
     autosize.update($("#compose-textarea"));
+
+    // While in preview mode we disables unneeded compose_control_buttons,
```

Comment:
nitpick - disables -> disable.

Reply:
I should replace disables with disable right?
I am just confused with nitpick - disables.

Reply:
> I should replace disables with disable right?

Yes.